internal/configuration/settings.go

```diff
@@ -266,10 +266,10 @@ func (s *Settings) handleDeleteEvent(obj interface{}) {
 	}
 }
 
-func (s *Settings) handleUpdateEvent(_ interface{}, obj interface{}) {
+func (s *Settings) handleUpdateEvent(_ interface{}, newValue interface{}) {
```

Comment:
optional nit: would go allow `handleUpdateEvent(_oldValue interface{}, newValue interface{})`?  It seems this is a callback so although we are not using the value it would be nice to make it clear what that unused value is.

Reply:
I had it like that, but Go does not recognize the underscore prefix as an unused parameter convention and linting calls it out. I much prefer the Elixir pattern where the name is preferred.